This patch fixes dist_clean target used to remove generated files before
packaging sources.

diff -urNad lazarus-0.9.26.1-0~/Makefile.fpc lazarus-0.9.26.1-0/Makefile.fpc
--- lazarus-0.9.26.1-0~/Makefile.fpc	2008-10-11 19:19:59.000000000 +0200
+++ lazarus-0.9.26.1-0/Makefile.fpc	2008-10-11 19:20:01.000000000 +0200
@@ -11,7 +11,7 @@
 
 [target]
 programs=lazarus startlazarus
-dirs=lcl packager/registration ideintf components
+dirs=lcl packager/registration ideintf components ide tools
 exampledirs=examples
 
 [compiler]
@@ -48,10 +48,6 @@
 .PHONY: lcl components packager/registration ideintf bigidecomponents ide idepkg idebig bigide starter lazbuilder tools all clean cleanide cleannogui purge examples install lazbuild
 
 #-----------------------------------------------------------------------------
-ide: 
-	$(MAKE) -C ide ide
-
-#-----------------------------------------------------------------------------
 idepkg:
 	$(MAKE) -C ide idepkg
 
@@ -82,8 +78,7 @@
 
 #-----------------------------------------------------------------------------
 tools: lcl
-	$(MAKE) -C tools
-	
+
 #-----------------------------------------------------------------------------
 examples: lcl components
 
@@ -88,7 +84,7 @@ tools: lcl
 examples: lcl components
 
 #-----------------------------------------------------------------------------
-all: packager/registration lcl ideintf components ide starter lazbuilder
+all: packager/registration lcl ideintf components bigidecomponents bigide starter lazbuilder tools examples
 
 #-----------------------------------------------------------------------------
 cleannogui:
